packages/xo-server/src/xo-mixins/xen-servers.js

```diff
@@ -11,13 +12,16 @@ import {
   forEach,
   isEmpty,
   isString,
+  pDelay,
```

Comment:
Import from `promise-toolbox` instead, let's try to shrink `utils` as much as we can.